references/cli/addon_test.go

```diff
@@ -421,16 +421,16 @@ func TestGenerateParameterString(t *testing.T) {
 }
 
 func TestNewAddonCreateCommand(t *testing.T) {
-	cmd := NewAddonCreateCommand()
+	cmd := NewAddonInitCommand()
 	cmd.SetArgs([]string{})
 	err := cmd.Execute()
 	assert.ErrorContains(t, err, "required")
 
-	cmd.SetArgs([]string{"--chart", "a", "--helm-repo-url", "https://some.com", "--version", "c"})
+	cmd.SetArgs([]string{"--chart", "a", "--helm-repo", "https://some.com", "--chart-version", "c"})
 	err = cmd.Execute()
 	assert.ErrorContains(t, err, "required")
 
-	cmd.SetArgs([]string{"test-addon", "--chart", "a", "--helm-repo-url", "https://some.com", "--version", "c"})
```

Comment:
is this a break change?

Reply:
Yes, the parameters are adjusted. But this function is for humans, there are no automation tools or other things that uses this, so it won't affect much.

The CLI examples have been updated, so the user can refer to help.

Reply:
well, we're not release it in a formal release, so we can change it for now. But next time, we should keep the consistence